docs/tutorial.md

```diff
@@ -0,0 +1,241 @@
+
+```js
+// ...
+
+bot.once('spawn', function () {
+  bot.chat('hi!');
+  // `bot.chat()` method for sending message or commad to the chat.
+});
+```
+
+For full methods list see the [documentation](http://mineflayer.prismarine.js.org/#/api?id=methods)
+
+### Listening for an event
+
+Bot object has many useful built in [events](http://mineflayer.prismarine.js.org/#/api?id=events).
+You can listen for an event by using either `bot.on()` method or `bot.once()` method of the Bot object, wich takes the name of event and a function.
+To remove specific listener you can use `bot.removeListener()` method.
+
+- `bot.on(eventName,listener)`
+  Adds the `listener` function to the end of the listeners array for the event named `eventName`. No checks are made to see if the listener has already been added. Multiple calls passing the same combination of `eventName` and `listener` will result in the `listener` being added, and called, multiple times.
+- `bot.once(eventName,listener)`
+  Adds a one-time listener function for the event named `eventName`. The next time `eventName` is triggered, this listener is removed and then invoked. Basically it'll run one time for the first event.
+- `bot.removeListener(eventName,listener)`
+  Removes the specified listener from the listener array for the event named eventName.
+
+Not only bot object, [`Chest`](http://mineflayer.prismarine.js.org/#/api?id=mineflayerchest), [`Furnace`](http://mineflayer.prismarine.js.org/#/api?id=mineflayerfurnace), [`Dispenser`](http://mineflayer.prismarine.js.org/#/api?id=mineflayerdispenser), [`EnchantmentTable`](http://mineflayer.prismarine.js.org/#/api?id=mineflayerenchantmenttable), [`Villager`](http://mineflayer.prismarine.js.org/#/api?id=mineflayervillager) object also has its own events!
+
+Examples :
+
+#### Echo Bot
+
+In here we're listening [`chat`](http://mineflayer.prismarine.js.org/#/api?id=quotchatquot-username-message-translate-jsonmsg-matches) event, and resend the message.
+[`chat`](http://mineflayer.prismarine.js.org/#/api?id=quotchatquot-username-message-translate-jsonmsg-matches) event only emitted when a player chats publicly.
+
+```js
+const mineflayer = require('mineflayer');
+const options = {...}
+const bot = mineflayer.createBot(options);
+
+function echo(username, message) {
+  if (username === bot.username) return;
+  bot.chat(message);
+}
+
+bot.on('chat', echo); // Only emitted when a player chats publicly.
+```
+
+`bot.on('chat', echo);` will listend for [`chat`](http://mineflayer.prismarine.js.org/#/api?id=quotchatquot-username-message-translate-jsonmsg-matches) event, and execute `echo` function with argument of (username, message, translate, jsonMsg, matches) when [`chat`](http://mineflayer.prismarine.js.org/#/api?id=quotchatquot-username-message-translate-jsonmsg-matches) event triggered.
```

Comment:
> listend

Remove letter `d`

> with argument of (username, message, translate, jsonMsg, matches)

To: with arguments `username`, `message`, `translate`, `jsonMsg` and `matches`

> when `chat` event triggered.

Remove that or try to omit `chat` word

Reply:
Removing `chat` word

Reply:
Done